cabal-install/tests/IntegrationTests/exec/should_run/exit_with_failure_without_args.sh

```diff
@@ -0,0 +1,6 @@
+. ../common.sh
+
+# We should probably be using a .err file and should_fail,
```

Comment:
Yeah, I considered line-by-line regex comparison instead, but it's a bit noisy if you have to escape all the lines containing special characters (and where you *don't* want special treatment). FileCheck looks somewhat similar to that, but I think it's probably overkill for what we want.

Maybe we could add something simple here like line-based comparison, but with

```
   RE:this is a regex
   this is not
```

i.e. just using the a RE: prefix to designate regexes. (It's ulikely enough to appear in actual output that it wouldn't need to be "escapable".)